source/MaterialXTest/ShaderGen.cpp

```diff
@@ -1258,7 +1259,10 @@ TEST_CASE("Subgraphs", "[shadergen]")
     mx::DocumentPtr doc = mx::createDocument();
 
     mx::FilePath searchPath = mx::FilePath::getCurrentPath() / mx::FilePath("documents/Libraries");
-    loadLibraries({ "stdlib", "sxpbrlib" }, searchPath, doc);
```

Comment:
Not sure about this change. The vector initializer list syntax works for me and has worked for a long time I think. Why did you have to make this change?